deployment/v3/utils/prop_migrator/file_updater.py

```diff
@@ -0,0 +1,296 @@
+import csv
+import os
+import sys
+
+different_properties_csv = 'different_properties.csv'
+value_takes_priority_csv = 'knowledge/1.1.5.5-value-takes-priority.csv'
+lts_value_takes_priority_csv = 'knowledge/lts-Value-takes-priority.csv'
+new_directory = 'new-config/v1.2.0.1-B1'
```

Comment:
v1.2.0.1-B1  should be a dynamic value taken as part of migrator.csv, so that it can be passed to both comparator as well as updator.